lang/python/python3-drf-nested-routers/Makefile

```diff
@@ -1,17 +1,23 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=drf-nested-routers
-PKG_VERSION:=0.91
-PKG_RELEASE:=3
+PKG_VERSION:=0.92.0
+PKG_RELEASE:=1
 
 PYPI_NAME:=drf-nested-routers
-PKG_HASH:=46e5c3abc15c782cafafd7d75028e8f9121bbc6228e3599bbb48a3daa4585034
+PKG_HASH:=a73515a612ba16f7b200c938831b268932cf96da2f8ab18ef810fd53f9ed766f
 
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=LICENSE
 PKG_MAINTAINER:=Peter Stadler <peter.stadler@student.uibk.ac.at>
 
-include ../pypi.mk
+#include ../pypi.mk ### not working, use github instead:
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/alanjds/drf-nested-routers/archive/
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/$(PKG_NAME)$-$(PKG_VERSION)
```

Comment:
There is no host build of this package so I don't think `HOST_BUILD_DIR` is necessary.

Reply:
Thx, I will remove it (I copied it from pypi.mk)